### DISCUSSION TITLE
HaxeFlixel discord rich presence

### DISCUSSION THREAD

**Discussion by NikRetaNCAM** [HaxeFlixel/flixel#2486] [Q&A] [answered]
There's like literally no tutorial for that anywhere..
I know that there are APIs for this, but how to actually use them..

sorry for such dumb questions..

**Comment by RichardBray** [comment, marked as answer]
I'm actually not sure what question you're asking :)
Discord API docs HF API docs 🤷

**Reply by Geokureli** [reply]
yeah, flixel has nothing to do with discord, I doubt you'll find help here

**Reply by NikRetaNCAM** [reply]
ok sorry
